modules/machineset-aws-existing-placement-group.adoc

```diff
@@ -55,13 +55,15 @@ spec:
             availabilityZone: <zone> # <3>
             region: <region> # <4>
           placementGroupName: <placement_group> # <5>
+          placementGroupPartition: <placement partition number> # <6>
 # ...
 ----
 <1> Specify an instance type that link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/efa.html#efa-instance-types[supports EFAs].
 <2> Specify the `EFA` network interface type.
 <3> Specify the zone, for example, `us-east-1a`.
 <4> Specify the region, for example, `us-east-1`.
 <5> Specify the name of the existing AWS placement group to deploy machines in.
+<6> Specify the partition number of the existing AWS placement group to deploys machines in. 
 
 .Verification
```

Comment:
One line can be added to the `Verification` for `placementGroupPartition` like `placementGroupName`